### ISSUE TITLE
support multi-line fortunes with `%`

### ISSUE THREAD

**Issue by sewnie** [boneill02/fortune#1]
example:
```
%
The thing you have to remember about "hacker news" is that nothing posted there is
news and none of the people who frequent it are hackers.  "Web Developer with an
Unwarranted Sense of Self-Importance" just doesn't roll off the tongue quite as
readily as "hacker," just like "link-dump and self promotion" doesn't have the
same ring as "news." -- Kurt H. Maier
%
```

**Comment by bmoneill** [comment]
Fixed in b1b92c5

**Comment by sewnie** [comment]
it was that easy? wow!
